app/operations/story_operations.rb

```diff
@@ -142,7 +142,33 @@ def create_activity(story, current_user:)
     end
   end
 
-  class UpdateAll < BaseOperations::UpdateAll; end
+  class UpdateAll
+    include Dry::Monads[:result, :do]
+
+    def call(stories:, data:, current_user:)
+      stories = yield update_stories(
+        stories: stories,
+        data: data,
+        current_user: current_user
+      )
+
+      Success(stories)
+    rescue
+      Failure(false)
+    end
+
+    private
+
+    def update_stories(stories:, data:, current_user:)
+      updated_stories = stories.map do |story|
```

Comment:
Can you add a comment here for future reference? Like this:

```rb
# TODO: we should probably use a transaction here
```

We don't need to add the transaction now, since we just want it to keep working as before, but might be good to have it in a future PR.

Reply:
Adding
But should really be a transaction there? I mean, if the update of a single story fails, should it stop the update of the others?
maybe thats something to discuss later